src/pages/en/utokyo_account/mfa/yubikey-totp/index.md

```diff
@@ -0,0 +1,82 @@
+
+## Installing the “Yubico Authenticator” application
+
+First, you need to download the Yubico Authenticator application to your PC or smartphone and register the YubiKey.
+
+1. Please install the Yubico Authenticator app from the [download page](https://www.yubico.com/products/yubico-authenticator/).
+2. Connect your YubiKey by following the instructions of the Yubico Authenticator application.
+
+### Displaying the QR code
+
+You need to display the QR code to register the Yubico Authenticator application as a verification method for the multi-factor authentication of UTokyo Accounts. Since the method for displaying the QR code differs depending on the situations, details are explained separately.
+
+#### If you register Yubico Authenticator application as the first verification method
+
+If you have not registered any verification method and wish to register the Yubico Authenticator application as your first verification method, please follow the procedures below.
+
+1. Access the [Security info page](https://account.activedirectory.windowsazure.com/proofup.aspx?proofup=1&whr=utac.u-tokyo.ac.jp).
+2. The sign in page will appear (if you are not already signed in). Enter your UTokyo Account username (10-digit Common ID) and password and click “Sign in”.
+![](../first.png)
+3. Click “Next” in the “More information required” page.
+4. Click “I want to use a different authentication app” in the middle of the setup page.
+![](initial_other_auth_app.png)
+5. Follow the instructions until you reach the QR code page.
+![](initial_show_qr.png)
+
+#### If you have already registered the first verification method
+
+If you have already registered another verification method as your first one and wish to register the Yubico Authenticator application as an alternative verification method, please follow the procedures below.
+
+1. Click "Add method" in the [Security info page](https://account.activedirectory.windowsazure.com/proofup.aspx?proofup=1&whr=utac.u-tokyo.ac.jp).
+![](add_signin_method.png)
+2. For “Which method would you like to add?”, select “Authenticator app” and click “Add”.
+![](select_auth_app.png)
+3. Click “I want to use a different authentication app”.
+![](other_auth_app.png) 
+4. Follow the instructions until you reach the QR code page.
+![](show_qr.png)
+
+### Adding an account to Yubico Authenticator
+
+1. To add an account to the Yubico Authenticator application, please click on the “Configure Yubikey” in the upper right corner of the screen, and press the “Add account” displayed on the screen.
+![](add_account.png)
+1. Press the “Scan QR code” and scan the QR code according to the instructions on your screen, and press “Save”. <br>
+* If you are using a PC, the QR code displayed on the screen will be read in automatically and you do not need to scan the QR code with the camera of your PC. Therefore, when you do this process for the first time, it may ask you for the permission to record the screen.
```

Comment:
```suggestion
\* If you are using a PC, the QR code displayed on the screen will be read in automatically and you do not need to scan the QR code with the camera of your PC. Therefore, when you do this process for the first time, it may ask you for the permission to record the screen.
```

Reply:
`*`はmarkdown記法で箇条書きを作る特殊記号なので、直接`*`を書きたい場合には前にバックスラッシュ`\`をつけてエスケープしましょう。